extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -67,6 +67,9 @@ const SidebarEvents = {
   CLOSE: 'sidebarClose',
 };
 
+/** @private @const {Object<string, Object<string, Array<string>>>} */
+const EXCLUDE_FROM_SWIPE_CLOSE = {'input': {'type': ['range']}};
```

Comment:
Can we simplify this? This looks like a YAGNI smell - we're making it more complex than it needs to be. IMHO, in `excludeFromSwipeClose` you can just check for input type="range" and return.

Reply:
Yep, sounds good.  We can always add something like this in later if we decide to have more elements to exclude.  Thanks!